packages/circus-web-ui/src/pages/case-detail/processors/processor-types.ts

```diff
@@ -24,7 +24,11 @@ export const processorTypes = [
 
 export type ProcessorType = typeof processorTypes[number];
 
-export type ProcessorProgress = { value: number; label: string };
+export type ProcessorProgress = {
+  value: number;
+  label: string;
+  finished: boolean;
+};
```

Comment:
`finished: true` の時に label や value も更に独立して設定させて `{ value: 100, label: '', finished: true }` みたいにさせる意味はないと思います。そもそもそこで指定した `label` は結局どこにも表示されないわけで、「一体これは何に使うラベルなのか」と混乱してしまいます。

以下のようなシグネチャの方が自然では？

```suggestion
export type ProcessorProgress =
  | { value: number; label: string; }
  | { finished: boolean; };
```